.github/workflows/smoke.yml

```
@@ -0,0 +1,126 @@
+          - production
+          - staging
+
+jobs:
+  smoke-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        environment: [staging, production]
+        exclude:
+          - environment: production # Only run production on schedule, not on manual dispatch
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)